examples/with-typescript-graphql/lib/with-apollo.tsx

```diff
@@ -54,11 +73,25 @@ export default function withApollo(
 
   if (ssr || PageComponent.getInitialProps) {
     WithApollo.getInitialProps = async (ctx: WithApolloPageContext) => {
-      const { AppTree } = ctx
+      // Resolver context here is only set on server. For client-side,
+      // "/api/graphql" route creates and pass it to resolver functions.
+      let resolverContext: ResolverContext | undefined
+      // Keep the "isServer" check inline, so webpack removes the block
+      // for client-side bundle.
+      if (typeof window === 'undefined') {
+        if (!ctx.req || !ctx.res) throw new Error('never')
```

Comment:
```suggestion
```

Reply:
Thanks, but how do you think we can avoid the TypeScript error?
![Screen Shot 2020-04-09 at 9 34 54](https://user-images.githubusercontent.com/217530/78847322-6b52a100-7a49-11ea-80ea-6ec33c5a426e.png)

Reply:
With `ctx.req!` :+1: 

It will always be defined in this case.